contributors.md

```diff
@@ -1,4 +1,4 @@
-# Contributors to OpenRCT2
+﻿# Contributors to OpenRCT2
```

Comment:
You've accidentally introduced a byte-ordering mark to this file. Please revert this change.